### PR TITLE
Changes/fixes for planners

### PR DESCRIPTION
**Pull request by Jacenty00**
- Use arena agent with observation packer
- spacial horizon for cadrl
- add mpc to .rosinstall

### CODE REVIEW

File: arena_bringup/launch/testing/move_base/move_base_arena.launch [hunk 1]
```diff
@@ -1,19 +1,27 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <launch>
     <!-- <arg name="speed" default="0.22"/> -->
-    <arg name="model" default="turtlebo3_burger"/>
+    <arg name="model" default="turtlebo3_burger" />
```
Comment (ReykCS) [inline arena_bringup/launch/testing/move_base/move_base_arena.launch:4]: turtlebot3_burger is not a name of one of our robots, we should use burger here

Comment (Jacenty00) [reply arena_bringup/launch/testing/move_base/move_base_arena.launch]: Right! Changed it to burger.

**Review by ReykCS — APPROVED** [review]

**Comment by Jacenty00** [comment]
Added launch files for crowdnav and sarl
Added argument for flatland vizualisation in the main launch file

File: arena_bringup/launch/start_arena_flatland.launch [hunk 2]
```diff
@@ -8,7 +8,7 @@
   <arg name="agent_name" default="$(arg model)"     doc = "DRL agent name to be deployed" unless="$(eval arg('local_planner') != 'rosnav')"/>
   <arg name="task_mode" default="random" doc="[random, manual, staged, scenario]" />
   <arg name="visualization" default="rviz" doc="[rviz, flatland]" />
-  
+  <arg name="show_viz" default="true" doc="Enables flatland visualization" />
```
Comment (ReykCS) [inline arena_bringup/launch/start_arena_flatland.launch:11]: not needed anymore

**Review by ReykCS — APPROVED** [review]

**Pull request merged**